CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v0.9.0
+
+## Added
```

Comment:
We should add an entry about [the aarch64 specific `create_vm`](https://github.com/rust-vmm/kvm-ioctls/commit/363ac11884c7c3ba9964a61e8079b716b0ee31e7#diff-8b686fc2f6412cd60251eb4adf6d8bbe3e3b62fa8a48f394a4ff861f1d2a291bR389) as well.

Reply:
Done.